src/components/NavBar/NavBar.tsx

```diff
@@ -22,7 +22,7 @@ function NavBar() {
 	const [isLoggedIn, setLogIn] = useState(false);
 
 	return (
-		<div className='w-full flex items-center justify-between p-4'>
+		<div className='w-full flex items-center justify-between p-4 fixed top-0 bg-white z-10'>
```

Comment:
interesting - why did you need a z-index - just curious as i've never used it before!